src/Iteration.cpp

```diff
@@ -148,15 +148,29 @@ Iteration::flush()
     Writable *w = this;
     while( w->parent )
         w = w->parent;
-    Series* o = dynamic_cast<Series *>(w);
+    Series* s = dynamic_cast<Series *>(w);
 
-    meshes.flush(o->meshesPath());
-    for( auto& m : meshes )
-        m.second.flush(m.first);
+    //TODO warn if openPMD >= 1.1.0, mp is set and no meshes
```

Comment:
warning on a valid meshesPath without meshes is not necessary.

the only thing that 1.1.0 enforces is to throw an error if meshesPath is set and the path is invalid (group does not exist)

the same for particles below

Reply:
Brain was not fully working at 7AM yet.
When writing the TODO, I was mislead into thinking the path gets created unconditionally in ```Series```, but it does 6 lines below.